pkg/cri/server/container_create_test.go

```diff
@@ -416,3 +416,37 @@ func TestBaseRuntimeSpec(t *testing.T) {
 
 	assert.Equal(t, filepath.Join("/", constants.K8sContainerdNamespace, "id1"), out.Linux.CgroupsPath)
 }
+
+func TestRuntimeSnapshotter(t *testing.T) {
+	defaultRuntime := config.Runtime{
+		Type:        "io.containerd.runtime.v1.linux",
```

Comment:
Deprecated in favor of io.containerd.runc.v2

Reply:
I unset the runtime.Type since it's not irrelevant for the UT, thanks for the info!